### PR TITLE
Filter more mimetypes

### PR DESCRIPTION
**Pull request by LukasReschke**
There's no need to allow more than those defined mimetypes for images.

### CODE REVIEW

**Comment by rullzer** [comment]
yep!
LGTM!

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by ChristophWurst** [comment]
Regression: this broke contact images in Mail

This is shown if I try to open the image URL directly:
bildschirmfoto von 2016-10-06 22-52-05

cc @nickvergessen

File: apps/dav/lib/CardDAV/ImageExportPlugin.php [hunk 1]
```diff
@@ -87,6 +87,7 @@ function httpGet(RequestInterface $request, ResponseInterface $response) {
 
 		if ($result = $this->getPhoto($node)) {
 			$response->setHeader('Content-Type', $result['Content-Type']);
+			$response->setHeader('Content-Disposition', 'attachment');
```
Comment (ChristophWurst) [inline apps/dav/lib/CardDAV/ImageExportPlugin.php]: commenting out this line shows contacts images again.

**Comment by nickvergessen** [comment]
Ref https://github.com/nextcloud/mail/issues/131

**Comment by Maulwurf16** [comment]
I think this may have broken even more images on different places in server:

unbenannt

In the versions App - now every non-usual ending (as walletx here) is shown as a broken image. The image url is as follows:

index.php/apps/files_versions/preview?file=%2FEnpass%2Fsync_default.walletx&version=1474875142

with absolut no entry in any log

**Comment by LukasReschke** [comment]
@Maulwurf16: This is unrelated. Can you file a new issue? THX.

**Pull request merged**